src/mailadm/bot.py

```diff
@@ -62,7 +62,13 @@ def ac_incoming_message(self, message: deltachat.Message):
                 logging.debug("ignoring message, it's just admins discussing in a support group")
         else:
             chat = message.create_chat()
-            if chat.is_group():
+            if chat.is_protected() and not message.is_encrypted():
+                sender = message.get_sender_contact().addr
+                logging.warning("The bot doesn't trust %s, please re-add them to admin group" %
+                                (sender,))
+                message.chat.send_text("I didn't see %s being added to this group - can someone who"
```

Comment:
True. I'll think of something.

Reply:
I decided to create a new unverified group to tell the admin group members that the admin group is broken.

I also mitigated one corner case in which this can lead to an AttributeError (if someone in the group quote-replies to the group creation message, because then the bot thinks it's a support group).

I don't like this solution very much; it basically adds one more chat type which the bot needs to handle, thereby further increasing complexity. Maybe just leaving the recovery group directly after creating it would be better, what do you think?